extensions/wikia/AdEngine/js/slot/exitstitial.js

```diff
@@ -47,23 +57,28 @@ require([
 
 				$modalBody.append($modalText).append($modalAd).append($modalSkip);
 
-				// Show modal
+				// Load modal, but don't show its content yet
 				$modal = $.showModal(modalTitle, $modalBody, {
 					id: modalId,
 					width: modalWidth
-				});
+				}).css('opacity', 0);
 
-				// Show ads
+				// Once the ad loads, show the modal
 				$modalAd.html($('<div></div>').html($ad));
-				window.adslots2.push(adSlot);
+				win.adslots2.push({
+					slotName: adSlot,
+					onError: skipAd,
+					onSuccess: showModal
+				});
+				setTimeout(showModal, 1000); // Show modal after a second even if the ad still loads
```

Comment:
You can move the comment to the line above so it's consistent with another `setTimeout()` and its comment below :japanese_ogre: